bindings/rust/s2n-tls-sys/build.rs

```diff
@@ -154,20 +154,19 @@ fn build_vendored() {
 fn builder(libcrypto: &Libcrypto) -> cc::Build {
     let mut build = cc::Build::new();
 
+    let includes = [&libcrypto.include, "lib", "lib/api"];
     if let Ok(cflags) = std::env::var("CFLAGS") {
         // cc will read the CFLAGS env variable and prepend the compiler
         // command with all flags and includes from it, which may conflict
         // with the libcrypto includes we specify. To ensure the libcrypto
         // includes show up first in the compiler command, we prepend our
         // includes to CFLAGS.
```

Comment:
nit: The comment is a bit out of date now, if you think it'd be good to update it:
```suggestion
        // cc will read the CFLAGS env variable and prepend the compiler
        // command with all flags and includes from it, which may conflict
        // with the includes we specify. To ensure that our includes show
        // up first in the compiler command, we prepend them to CFLAGS.
```